src/Dependencies/Microsoft.DiaSymReader/ISymUnmanagedAsyncMethod.cs

```diff
@@ -12,13 +12,13 @@ namespace Microsoft.DiaSymReader
     public interface ISymUnmanagedAsyncMethod
     {
         [PreserveSig]
-        int IsAsyncMethod(out bool value);
+        int IsAsyncMethod([MarshalAs(UnmanagedType.Bool)]out bool value);
```

Comment:
Is it possible to remove ```[PreserveSig]``` on these methods? It's certainly desirable to make it easier for managed callers. There'd be two reasons not to do that:
1. We are trying to match another interop definition precisely (or this _is_ the interop assembly for others to consume)
2. The HRESULTs returned are non-standard (i.e. anything other than the usual S_OK, E_INVALIDARG, E_OUTOFMEMORY, E_POINTER, etc.)